models/models.reviews.js

```diff
@@ -20,25 +20,60 @@ exports.fetchReviews = (
   sort_by = "created_at",
   order = "DESC"
 ) => {
-  let categoryAdd = "";
+  const values = [];
+  const reviewColumns = [
+    "title",
+    "designer",
+    "owner",
+    "review_img_url",
+    "review_body",
+    "category",
+    "created_at",
+    "votes",
+  ];
 
-  if (category.length) {
-    categoryAdd = `WHERE category LIKE '%${category}%'`;
+  const categoryList = [
+    "",
+    "euro game",
+    "social deduction",
+    "dexterity",
+    "childrens games",
+  ];
```

Comment:
I would be hesitant to hard code these as you are using different devData, my first thought would be to pull the categories from the database and make sure you get a match. There might be an easier way but I will have to get back to you!